build/flaws.js

```diff
@@ -317,17 +317,22 @@ function injectPreWithHTMLFlaws(level, doc, $, rawContent) {
     // Because Cheerio doesn't support selectors like `pre + code` we have to
     // manually (double) check that the parent really is a `<pre>` tag.
     if ($pre.length && $pre.get(0).tagName === "pre") {
-      addFlaw($pre);
+      addCodeTagFlaw($pre);
+      flawsFound = true;
     }
   });
 
+  if (!flawsFound) {
+    // more checks
```

Comment:
Can you please fix this. I'm just afraid that we forget what we discussed. 
Can you write a bit more about what the potential here. Perhaps linking to some issue discussions and stuff. In fact, feel free to file an issue if you have to and type something like:
```
// TODO: Bla bla bla
// See https://github.com/mdn/yari/issues/9999
```

Reply:
I realized that the commend above `flawsFound` declaration was too easy to miss, so I removed `flawsFound` variable and used `doc.flaws.bad_pre_tags.length === 0` to signify there are no flaws found yet.